worlds/sc2wol/Options.py

```diff
@@ -231,6 +258,131 @@ class MaxNumberOfUpgrades(Range):
     default = -1
 
 
+
+class IncludeMutations(Range):
+    """Determines how many of the 3 mutations for the 7 units that have them can appear."""
+    display_name = "Include Mutations"
+    range_start = 0
+    range_end = 3
+    default = 1
```

Comment:
The thing is if we'd want to use the limitation that's use for Terran units instead

Reply:
Technically those already work together, since the mutations add the unpicked upgrades to `excluded_items` and the max upgrades option removes from the inventory directly. I agree there's overlap in function though, once Zerg units get more upgrades it makes sense to remove this option.

Reply:
One of the things we have to think about is how to classify Ravager